content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,19 @@
+title: "A Statement on Richard Stallman"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+_A Statement from Mixxx on Richard Stallman's return to the FSF._
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a partial acknowledgement of the [harm he's caused over multiple decades](https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88), including creating a sexually hostile work and learning environment, sexual harassment, and abuse of power.
+Loss of his position in the community was only the barest of consequences, but his resignation came as a relief for many in the community who have suffered his bad behavior.
+
+While Mixxx is not officially associated with the Free Software Foundation,
+there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
```

Comment:
Moreover, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 *or later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
Therefore the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.

Reply:
added (slightly tweaked)